docker/daemon/daemon_types.go

```diff
@@ -41,7 +41,7 @@ type schema2Manifest struct {
 
 // Based on github.com/docker/docker/image/image.go
 // MOST CONTENT OMITTED AS UNNECESSARY
-type image struct {
+type dockerImage struct {
```

Comment:
do we plan to have other `*Image` types? just `image` was nice to have. Not a blocker, feel free to skip this comment.